js/chart-config.js

```diff
@@ -22,11 +22,11 @@ var charts = {
   // ================== Labour Tree Map ===================== //
   st_population : {
     chart_options : {
-      title : 'Population Pyramid of Myanmar',
+      title : 'Population pyramid of Myanmar',
```

Comment:
@DragonVirus this JS file only contains JS logic for the burmese dashboard? If so, it should be renamed to include some prefix or sufix indicating that it applies only to a specific template